### PR TITLE
Timeline feature does not respect security constraint set by "add-group-repo" kie-cli command

### PR DESCRIPTION
**Pull request by ederign**
Depends on : https://github.com/droolsjbpm/kie-uberfire-extensions/pull/13

### CODE REVIEW

File: kie-wb-common-screens/kie-wb-common-social-home-page/kie-wb-common-social-home-page-backend/src/main/java/org/kie/workbench/common/screens/social/hp/security/SocialEventOUConstraint.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+    private RepositoryServiceImpl repositoryService;
+
+    private User identity;
+
+    private Collection<OrganizationalUnit> authorizedOrganizationalUnits = new HashSet<OrganizationalUnit>();
+
+    @Inject
+    public SocialEventOUConstraint( final OrganizationalUnitService organizationalUnitService,
+                                            final AuthorizationManager authorizationManager,
```
Comment (manstis) [inline kie-wb-common-screens/kie-wb-common-social-home-page/kie-wb-common-social-home-page-backend/src/main/java/org/kie/workbench/common/screens/social/hp/security/SocialEventOUConstraint.java]: Formatting's a bit "off".. but not a blocker for the PR.

File: kie-wb-common-screens/kie-wb-common-social-home-page/kie-wb-common-social-home-page-backend/src/main/test/org/kie/workbench/common/screens/social/hp/security/SocialEventRepositoryConstraintTest.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        socialEventRepositoryConstraint.init();
+
+        assertTrue( socialEventRepositoryConstraint.hasRestrictions( event ) );
+
+    }
+
+    @Test
+    public void hasNoRestrictionsTest() throws Exception {
+
+        returnRepo = repository;
```
Comment (manstis) [inline kie-wb-common-screens/kie-wb-common-social-home-page/kie-wb-common-social-home-page-backend/src/main/test/org/kie/workbench/common/screens/social/hp/security/SocialEventRepositoryConstraintTest.java]: What's the significance of this? I can't see it's used?

File: kie-wb-common-screens/kie-wb-common-social-home-page/kie-wb-common-social-home-page-client/src/main/test/org/kie/workbench/common/screens/social/hp/client/homepage/DefaultSocialLinkCommandGeneratorTest.java [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+import static org.mockito.Mockito.*;
+
+@RunWith( MockitoJUnitRunner.class )
+public class DefaultSocialLinkCommandGeneratorTest {
+
+
+    DefaultSocialLinkCommandGenerator generator;
+    private ParameterizedCommand<LinkCommandParams> command;
+    private boolean hasAccessRightsForFeature = false;
```
Comment (manstis) [inline kie-wb-common-screens/kie-wb-common-social-home-page/kie-wb-common-social-home-page-client/src/main/test/org/kie/workbench/common/screens/social/hp/client/homepage/DefaultSocialLinkCommandGeneratorTest.java]: What's the significance of this? I cannot see it's ever "read"

**Comment by manstis** [comment]
+1 to merge.

**Comment by porcelli** [comment]
https://github.com/droolsjbpm/kie-wb-common/commit/619e18b60fefa023fa1260224a9551c4df06fade

**Pull request closed without merging**